doc/user/layouts/shortcodes/mysql-direct/ingesting-data/use-aws-privatelink.html

```diff
@@ -2,127 +2,136 @@
-      );
-      ```
+```mzsql
+CREATE CONNECTION privatelink_svc TO AWS PRIVATELINK (
+SERVICE NAME 'com.amazonaws.vpce.<region_id>.vpce-svc-<endpoint_service_id>',
+    AVAILABILITY ZONES ('use1-az1', 'use1-az2', 'use1-az4')
+    );
+    ```
 
-    - Replace the `SERVICE NAME` value with the service name you noted [earlier](#b-optional-configure-network-security).
+    - Replace the `SERVICE NAME` value with the service name you noted
+    [earlier](#b-optional-configure-network-security).
 
     - Replace the `AVAILABILITY ZONES` list with the IDs of the availability
-      zones in your AWS account. For in-region connections the availability
-      zones of the NLB and the consumer VPC **must match**.
+    zones in your AWS account. For in-region connections the availability
+    zones of the NLB and the consumer VPC **must match**.
 
-      To find your availability zone IDs, select your database in the RDS
-      Console and click the subnets under **Connectivity & security**. For each
-      subnet, look for **Availability Zone ID** (e.g., `use1-az6`),
-      not **Availability Zone** (e.g., `us-east-1d`).
+    To find your availability zone IDs, select your database in the RDS
+    Console and click the subnets under **Connectivity & security**. For each
+    subnet, look for **Availability Zone ID** (e.g., `use1-az6`),
+    not **Availability Zone** (e.g., `us-east-1d`).
 
     ↔️ **Cross-region connections**
 
     To connect to an AWS PrivateLink endpoint service in a **different region** to
     the one where your Materialize environment is deployed:
 
-      ```mzsql
-      CREATE CONNECTION privatelink_svc TO AWS PRIVATELINK (
-        SERVICE NAME 'com.amazonaws.vpce.us-west-1.vpce-svc-<endpoint_service_id>',
-        -- For now, the AVAILABILITY ZONES clause **is** required, but will be
-        -- made optional in a future release.
-        AVAILABILITY ZONES ()
+    ```mzsql
+    CREATE CONNECTION privatelink_svc TO AWS PRIVATELINK (
+    SERVICE NAME 'com.amazonaws.vpce.us-west-1.vpce-svc-<endpoint_service_id>',
+      -- For now, the AVAILABILITY ZONES clause **is** required, but will be
+      -- made optional in a future release.
+      AVAILABILITY ZONES ()
       );
       ```
 
-    - Replace the `SERVICE NAME` value with the service name you noted [earlier](#b-optional-configure-network-security).
+      - Replace the `SERVICE NAME` value with the service name you noted
+      [earlier](#b-optional-configure-network-security).
 
-    - The service name region refers to where the endpoint service was created.
+      - The service name region refers to where the endpoint service was created.
       You **do not need** to specify `AVAILABILITY ZONES` manually — these will
       be optimally auto-assigned when none are provided.
 
-1. Retrieve the AWS principal for the AWS PrivateLink connection you just
-created:
+      - For Kafka connections, it is required for
```

Comment:
Q: Should this be here? I thought this content was when we directly use our MySQL connector and not Kafka?
...
we do have  https://preview.materialize.com/materialize/32414/ingest-data/mysql/mysql-debezium/#c-create-a-source and https://preview.materialize.com/materialize/32414/ingest-data/postgres/postgres-debezium/ pages (although we don't talk about private links on those pages).

Reply:
Maybe mention on https://preview.materialize.com/materialize/32414/sql/create-connection/#aws-privatelink